src/components/ConnectionLayout.tsx

```diff
@@ -20,15 +20,15 @@ type ConnectionLayoutProps = {
     /** Header title for the connection */
     headerTitle: TranslationPaths;
 
+    /** The subtitle to show in the header */
+    headerSubtitle?: string;
+
     /** React nodes that will be shown */
     children?: React.ReactNode;
 
     /** Title of the connection component */
     title?: TranslationPaths;
 
-    /** Subtitle of the connection */
-    subtitle?: TranslationPaths;
```

Comment:
This wasn't used anywhere, so I'm removing it. Let me know if I'm wrong

Reply:
Makes sense and we can add it again if required later. I don't see any of my/other screens also using this.